### DISCUSSION TITLE
Dynamically Wire Up

### DISCUSSION THREAD

**Discussion by nfplee** [vuejs/petite-vue#93] [General]
See the following JSFiddle for an example:

https://jsfiddle.net/85rebpaL/

Notice how when you click the update button it works fine for Alpine.js but not for petite-vue.

**Comment by nooooooom** [comment]
`petite-vue` does not have the logic to automatically initialize after the browser is loaded, so we need to deal with the timing of introduction.

The [documentation](https://github.com/vuejs/petite-vue#usage) recommends that you use `defer`, or you can move the scripts to end of `<body>`.

**Reply by nfplee** [reply]
Thanks but this doesn't address my issue. I've updated my JSFiddle to make my intentions clearer. Imagine say you did an Ajax request to the server which returns the HTML that is set against the container element.

**Reply by nooooooom** [reply]
Oh! I understand, Alpine.js will observer for mutations.

**Comment by niutech** [comment]
You can manually mount Petite Vue app inside `onclick` event handler: `PetiteVue.createApp().mount()`, see [demo](https://jsfiddle.net/9nxoz7sq/1/).
